### PR TITLE
Fixes #20478: metrics are still present in wizard, and wizard doesn't end

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/20478

### CODE REVIEW

**Comment by ncharles** [comment]
This PR removes the metrics totally
However, it suffers from a massive problem as the message "Your account aaaaaa has been linked to your Rudder!" at the end is wrong. File is not yet saved when we display it, and I'm wondering why there is such a message there before saving

File: webapp/sources/rudder/rudder-web/src/main/elm/onboarding/sources/View.elm [hunk 1]
```diff
@@ -74,10 +74,10 @@ view model =
           Account s ac ->
             case s of
               Completed -> ( completeClass , completeIcon , "Your account "++ ac.username ++ " has been linked to your Rudder!"  )
```
Comment (ElaadF) [inline webapp/sources/rudder/rudder-web/src/main/elm/onboarding/sources/View.elm:76]: Change this line to 
```Elm
Completed -> ( completeClass , completeIcon , "Your account "++ ac.username ++ " will be linked to your Rudder server"  )
```

**Comment by ElaadF** [comment]
Metrics section is removed :heavy_check_mark: 
image

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by ElaadF — APPROVED** [review]

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**